### DISCUSSION TITLE
How to uninstall from MacOS

### DISCUSSION THREAD

**Discussion by joe-stroup** [LMS-Community/slimserver#1113] [General]
I have the LMS installed on a Mac, and I want to remove it and re-install it on a different computer in my home.

I installed LMS about a month ago - and I don't recall that it was called Lyrion at that time - I'm wondering if there's been some changes.

For example, I don't have the Lyrion app - rather it shows up at the bottom of the list in Settings.  There's not application for it in the Applications folder.

Can someone please guide me to do a clean uninstall?  Thanks!

**Comment by michaelherger** [comment]
Heh... that's actually a good question, and I'm surprised it hadn't come up before! Basically check Settings/Information for the three paths for Cache, Settings, and Logs. It's something like:

cache: /Users/{user}/Library/Caches/Squeezebox
prefs: /Users/{user}/Library/Application Support/Squeezebox
logs: /Users/{user}/Library/Logs/Squeezebox

Stop LMS, remove it from the preferences, delete those three folders.

**Reply by joe-stroup** [reply]
Thank you Michael - 

Are you sure only those three folders are all that need to be removed?  This display suggest there might be other components:

![Screenshot 2024-06-06 at 9 18 22 AM](https://github.com/LMS-Community/slimserver/assets/91474707/0c6dab80-3f8c-49ad-a106-3868744d0126)
